src/components/AuthorAvatarIcon.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { PropTypes } from 'react';
+import {
+  Avatar,
+} from 'material-ui';
+import ReactTooltip from 'react-tooltip';
+
+const styles = {
+  avatar: {
+    flexBasis: 'auto',
+    margin: '3px',
+    marginLeft: '20px',
+    paddingTop: '10px',
+    verticalAlign: 'bottom',
+  },
+};
+
+const AuthorAvatarIcon = ({ author, avatarUrl }) => {
+  if (avatarUrl) {
+    return (
+      <div
+        data-for="avatar"
```

Comment:
I think it might be better to change this to `data-for="author-avatar"`. If we ever added a tooltip to an `<Avatar />` component, we would likely use `data-for="avatar"`, which would clash with this div's current `data-for` value.